curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/american-british-translator.md

```diff
@@ -101,7 +101,7 @@ async (getUserInput) => {
 };
 ```
 
-The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English.
+The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English. NOTE: The `span` element should wrap the full time string, i.e. `<span class="highlight">10:30</span>`.
```

Comment:
@nhcarrigan Though not specified in the [docs](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-work-on-coding-challenges.md), notes should be only a separate line.  The documentation does specify the format of the note (see below).  I also am suggesting to change the word "full" to "entire".

```suggestion
The `/api/translate` route should handle the way time is written in American and British English. For example, ten thirty is written as "10.30" in British English and "10:30" in American English.

**Note:** The `span` element should wrap the entire time string, i.e. `<span class="highlight">10:30</span>`.
```

Reply:
I think we could add the mention of a new line, because this is mentioned:
> - Additional information in the form of a note should be formatted `**Note:** Rest of note text...`

https://github.com/freeCodeCamp/freeCodeCamp/blame/master/docs/how-to-work-on-coding-challenges.md#L279